src/deepsparse/pipeline.py

```diff
@@ -175,6 +176,8 @@ def __init__(
 
         self.onnx_file_path = self.setup_onnx_file_path()
         self.engine = self._initialize_engine()
+        self._inference_timing = None
+        self.timer = TimingBuilder()
```

Comment:
What about just having a `self.call_times: List[TimingBuilder] = []`, and then `__call__` can append/extend this list?

Then in server we can just dip into this call_times list.

Having a inference timing schema feels overkill unless i'm missing some use case for it!